ocs_sample_library_preview/Securable.py

```diff
@@ -0,0 +1,210 @@
+from ocs_sample_library_preview.Security.AccessRights import AccessRights
+from jsonpatch import JsonPatch
+from typing import Any
+
+from .BaseClient import BaseClient
+from .Security.AccessControlList import AccessControlList
+from .Security.Owner import Owner
+
+class Securable(object):
+    """
+    Client for interacting with the access control of a collection
+    """
+
+    def __init__(self, client: BaseClient, collection: str):
+        """
+        :param client: base client that handles auth and base routing
+        :param acl_path: The access control path of the object
+        """
+        self.__tenant = client.tenant
+        self.__uri_api = client.uri_API
+        self.__base_client = client
+        self.__collection = collection
+
+        self.__setPathAndQueryTemplates()
+
+    def getDefaultAccessControl(self, namespace_id: str) -> AccessControlList:
+        """
+        Get a collection's default access control list
+        """
+        response = self.__base_client.request(
+            'get', self.__default_collection_acl_path.format(
+                tenant_id=self.__tenant,
+                namespace_id=namespace_id))
+        self.__base_client.checkResponse(
+            response, f'Failed to get collection access control, {self.__collection}.')
+        result = AccessControlList.fromJson(response.json())
```

Comment:
I would probably just do `return AccessControlList.fromJson....` but this is OK.

Reply:
I will leave this for now since we do this in other parts of the library (see Streams for example).